### PR TITLE
1 cloudwatch log

### PR DESCRIPTION
**Pull request by speedcm**
The Hedwig application can now forward logs to CloudWatch with the implementation of GitHub issue 680.

To ensure logs have a proper retention time period, the updated Terraform scripts must be applied to the environment (e.g. Staging, Production) prior to the rollout of this fix.

This feature gives us the ability to also adjust the log levels reported in CloudWatch by adjusting the values of the Release Pipeline “Logging.*” variables (Logging.LogLevel.Default, Logging.LogLevel.Microsoft & Logging.LogLevel.System).  

This feature also gives us the ability to instrument our own application logger logs:

   Example: logger.LogInformation("Detected environment " + environment);

The application CloudWatch logs are accessible via:

* Browse to https://us-east-2.console.aws.amazon.com/cloudwatch/home?region=us-east-2#
* Click Log groups
* Click on /aws/elasticbeanstalk/ece-component-stage-stage/app-log

This fix has been implemented for Hedwig UAT.  It still needs to be rolled out for Hedwig Staging & Production.

This fix has not been implemented for Winged Keys.  

To test this fix, the azure-pipelines.yml file was updated to allow for packaging artifacts for a PR build.  This was set back after testing was completed.

### CODE REVIEW

**Comment by speedcm** [comment]
This PR is ready to be reviewed.

**Comment by speedcm** [comment]
Looks like the change that made it all work was logging.AddAWSProvider(context.Configuration.GetAWSLoggingConfigSection());

File: src/Hedwig/Program.cs [hunk 1]
```diff
@@ -52,7 +55,7 @@ public static void Main(string[] args)
 				var environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
```
Comment (jlhogan) [inline src/Hedwig/Program.cs]: this line should not be here -- we switched the way we manage the concept of env with env vars: https://github.com/ctoec/ecis-experimental/blob/master/docs/configuration/2020_01_07_custom-multi-env.md

could you just remove this line as a rider in the PR? The defn for environment should come from the declaration on line 46. 

ty!! @speedcm

Comment (speedcm) [reply src/Hedwig/Program.cs]: I removed line - var environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");

**Pull request merged**